backend/app/core/security.py

```
@@ -0,0 +1,144 @@
+"""
+Security utilities for SEPE Templates Comparator.
+
+This module provides JWT token handling, password hashing, and authentication
+utilities for the application.
+"""
+
+from datetime import datetime, timedelta
+from typing import Optional, Union
+import jwt
+from passlib.context import CryptContext
+from passlib.hash import bcrypt
```

Flake8 (7.3.0)

[error] 'passlib.hash.bcrypt' imported but unused

(F401)